db/upgrade.php

```diff
@@ -80,5 +80,14 @@ function xmldb_tool_crawler_upgrade($oldversion) {
         upgrade_plugin_savepoint(true, 2019072600, 'tool', 'crawler');
     }
 
+    if ($oldversion < 2019100100) {
+        $table = new xmldb_table('tool_crawler_url');
+        $field = new xmldb_field('priority', XMLDB_TYPE_NUMBER, '10', null, false, false, TOOL_CRAWLER_PRIORITY_DEFAULT);
+        if (!$dbman->field_exists($table, $field)) {
+            $dbman->add_field($table, $field);
```

Comment:
this missing a companion change to install.xml

Also I reckon sure this may cause perf issues on a big moodle. I think this will require a join index on both priority and needscrawl